.gitignore

```diff
@@ -1 +1,2 @@
 .idea
+results/*
```

Comment:
Also need to add log/*

Reply:
And logerrors.o, logerrors.so. Although this does not apply to changes :)

Reply:
sure thing